### DISCUSSION TITLE
请求退款一年

### DISCUSSION THREAD

**Discussion by seagull2024** [getlantern/lantern-forum-cn#1012] [General]
我续费一年后，jxtxxxzl#163.com 显示IP不在白名单。也没跳转成功付款，所以又付了一次，才发现付了两次。我是帮公司续的，请退我一年的钱。

**Comment by MaxMKMax** [comment]
@seagull2024 您好. 好的, 请您提供我支付单据和您的账号邮箱. 我帮您提交问题给他们退款錯誤付款.

Your email: jxtxxxzl @ 163 .com?

**Reply by seagull2024** [reply]
> @seagull2024 您好. 好的, 请您提供我支付单据和您的账号邮箱. 我帮您提交问题给他们退款錯誤付款.
> 
> Your email: jxtxxxzl @ 163 .com?

![微信图片_20241101105844](https://github.com/user-attachments/assets/12d5eb69-e87a-44dd-96f9-71acf3a653ca)
![微信图片_20241101105839](https://github.com/user-attachments/assets/d9d0c1a0-21da-446d-98f2-d2f75dbb2c2a)
这两个是我付款记录，帐号是：jxtxxzl@163.com

**Reply by seagull2024** [reply]
什么时候帮我处理？

**Reply by seagull2024** [reply]
> @seagull2024 您好. 好的, 请您提供我支付单据和您的账号邮箱. 我帮您提交问题给他们退款錯誤付款.
> 
> Your email: jxtxxxzl @ 163 .com?
帮我提交上去了，大概什么时候能退呢？谢谢。
